api/v1beta2/foundationdbcluster_types.go

```diff
@@ -1718,6 +1718,12 @@ type RoutingConfig struct {
 	// latest stable version of FoundationDB.
 	UseDNSInClusterFile *bool `json:"useDNSInClusterFile,omitempty"`
 
+	// DefineDNSLocalityFields determines whether to define pod DNS names on pod
```

Comment:
```suggestion
	// DefineDNSLocalityFields determines whether to define Pod DNS names on Pod
```

Reply:
We may need a breakout session on whether "pod" is a proper noun for style purposes. 😁

Reply:
Sounds reasonable, even the official docs are inconsistent 🙄 https://kubernetes.io/docs/concepts/workloads/pods/